client/src/main/kotlin/io/spine/examples/chatspn/App.kt

```diff
@@ -37,25 +23,38 @@ import androidx.compose.ui.unit.dp
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+package io.spine.examples.chatspn
```

Comment:
I think, and that's a general idea, we should state explicitly this is a client-level code. Therefore, I would have `.client` or `.desktop` package here.